types/parcel/index.d.cts

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
```

Comment:
Should this file be built during `build-types.js` and gitignored? It seems to be an exact copy of parcel/index.d.ts

Reply:
Files differ in the way the exports are handled. We might be able to maintain a single file, and do build-time transformations, but I would leave this for another PR.

Reply:
Ah the files differ - nevermind then it's ok to keep this file